.github/workflows/ci.yml

```diff
@@ -129,16 +127,16 @@ jobs:
       run: composer require --dev dereuromark/composer-prefer-lowest && vendor/bin/validate-prefer-lowest -m
 
     - name: Submit code coverage
-      if: matrix.php-version == '8.0'
-      uses: codecov/codecov-action@v1
+      if: matrix.php-version == '8.1'
+      uses: codecov/codecov-action@v2
 
   testsuite-windows:
     runs-on: windows-2019
-    name: Windows - PHP 8.0 & SQL Server
+    name: Windows - PHP 8.1 & SQL Server
 
     env:
-      EXTENSIONS: mbstring, intl, apcu, memcached, redis, pdo_sqlsrv
```

Comment:
I guess we'll have to wait for a new release to be made for this extension?

Reply:
Yup we need to wait for a stable release of memcached with PHP 8.1 support for Windows.